Trap Quest.inform/Inform/Extensions/Player/Exposure Addiction.i7x

```diff
@@ -22,4 +22,26 @@ This is the knickers can't be summoned with no panties fetish rule:
 		unless wearing-target is diaper or (wearing-target is no protection and diaper lover <= 0), rule fails.
 The knickers can't be summoned with no panties fetish rule is listed in the knickers wearability rules.
 
-No Panties Addiction ends here.
+
+topless permanent fetish is a number that varies.  topless permanent fetish is 0.
+Definition: yourself is fighting against no-bra:
+	if topless permanent fetish is 1 or there is a worn topless tattoo or the humiliation of the player >= 35000:
+		let C be a random worn bra;
+		if C is bra and C is not no protection, decide yes;
+	decide no.
+braless reminder is a number that varies. braless reminder is 0.
+A time based rule (this is the fighting against no-bra fetish rule):
+	if the player is fighting against no-bra:
+		if braless reminder <= 0:
+			let C be a random worn bra;
+			if C is bra, say "[bold type]You [one of][or]still [stopping]feel terrible wearing your [printed name of C][bold type]!  [one of][if there is a worn topless tattoo]Your heart aches and you[else]You[end if] want to free your [BreastDesc]. It's lowers your awareness and consumes your thought.[or][stopping][roman type][line break]";
```

Comment:
correct English would be "**It** lowers your awareness and consumes your **thoughts**".